qt/scientific_interfaces/ISISReflectometry/GUI/Runs/ISearcher.h

```diff
@@ -18,6 +18,7 @@ namespace CustomInterfaces {
 class SearcherSubscriber {
 public:
   virtual void notifySearchComplete() = 0;
+  virtual void notifyAutoreductionPaused() = 0;
```

Comment:
I'd like to avoid knowledge of autoreduction in the searcher. Could we add a different callback, say, `notifySearchFailed` instead? And then have an override in `RunsPresenter` which calls `notifyAutoreductionPaused` - and actually it probably only needs to do that if `isAutoreducing()` is true.